backend/src/packages/files/libs/helpers/get-url.ts

```diff
@@ -0,0 +1,18 @@
+function generateUrl(
+  template: string,
+  placeholders: Record<string, string>,
+): string {
+  let url = template;
+  for (const key in placeholders) {
+    if (Object.hasOwnProperty.call(placeholders, key)) {
```

Comment:
Maybe let's use `Object.hasOwn`, as it is a recommended over `hasOwnProperty()`?
```suggestion
    if (Object.hasOwn(placeholders, key)) {
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn

Reply:
hey, thanks, for suggestion - updated my code!

Reply:
now you can even remove the `Object.hasOwn()` condition, as `Object.keys()` iterates only over properties that are attached to the object itself

Reply:
yep, you are right - made changes in a hurry ;)